components/tools/OmeroWeb/test/integration/test_api_login.py

```diff
@@ -71,7 +71,7 @@ def test_base_url_versions_404(self):
         """
         Tests that the base url gives 404 for invalid versions
         """
-        version = '0'
+        version = '0.1'
```

Comment:
maybe better to access the api version from ``settings.py`` and add +1 for example so we do not have to be updating both